MAUI/NavigationDrawer/Getting-Started.md

```diff
@@ -7,17 +7,32 @@ control: NavigationDrawer
 documentation: ug
 ---
 
-# Getting Started with .NET MAUI Navigation Drawer (SfNavigationDrawer)
+# Getting Started with .NET MAUI Navigation Drawer
 
-This section explains how to add the [.NET MAUI NavigationDrawer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.SfNavigationDrawer.html) control. This section covers only the basic features needed to get started with Syncfusion .NET MAUI NavigationDrawer.
+This section guides you through setting up and configuring a [NavigationDrawer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.SfNavigationDrawer.html) in your .NET MAUI application. Follow the steps below to add a basic Tab View to your project.
```

Comment:
This section guides you through setting up and configuring a [NavigationDrawer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.NavigationDrawer.SfNavigationDrawer.html) in your .NET MAUI application. Follow the steps below to add a basic Navigation Drawer to your project.

Reply:
Yes, I have changed